tests/utils/plot_utils_test.py

```diff
@@ -591,15 +591,18 @@ def test_create_mantis_dir(
             new_mapping_df = pd.read_csv(
                 os.path.join(output_path, f"population{mask_suff}.csv"))
 
-            # 3.a. Assert that metacluster col equals the region_id col
-            metacluster_col = original_mapping_df[[f"{cluster_type}_meta_cluster"]]
+            # 3.a. Assert that appropratiate cluster col equals the region_id col
+            cluster_col = "cluster_id" if cluster_type == "cell"\
+                else f"{cluster_type}_meta_cluster"
+            metacluster_col = original_mapping_df[[cluster_col]].drop_duplicates()
             region_id_col = new_mapping_df[["region_id"]]
-            metacluster_col.eq(region_id_col)
+            assert np.equal(metacluster_col.values, region_id_col.values).all()
```

Comment:
Got any preference on validating on the values stored in DataFrame vs the DataFrame as a whole (includes column names, dtypes, and indexes)?

Reply:
These have different column names depending on whether it's the original mapping file or the new one generated for the mantis dir.

Reply:
oh okay, makes sense